spec.html

```diff
@@ -2910,56 +2910,56 @@
       <p>A <dfn id="super-reference">Super Reference</dfn> is a Reference that is used to represents a name binding that was expressed using the super keyword. A Super Reference has an additional thisValue component, and its base value component will never be an Environment Record.</p>
       <p>The following abstract operations are used in this specification to operate on references:</p>
 
-      <emu-clause id="sec-getbase" aoid="GetBase">
-        <h1 id="ao-getbase">GetBase ( _V_ )</h1>
+      <emu-clause id="sec-getbase" aoid="GetBase" oldids="ao-getbase">
```

Comment:
I see, this is what I get for going backward through the commits.

I can add oldids to ecmarkup, but I wonder if it's worth it over just adding an empty anchor. @domenic may have thoughts here too.